frontend/src/pages/Project/Members/index.tsx

```diff
@@ -30,6 +38,67 @@ export const ProjectMembers: React.FC<IProps> = ({ members, loading, onChange, r
         name: 'members',
     });
 
+    const currentUserRole = useMemo(() => {
+        if (!userData?.username) return null;
+        const member = members?.find(m => m.user.username === userData.username);
+        return member?.project_role || null;
+    }, [members, userData?.username]);
+
+    const isProjectOwner = useMemo(() => {
```

Comment:
```suggestion
    const isProjectOwner = userData?.username === project?.owner.username;
```
Don't need to use memoization here